src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arraySorceString = sourceString
+    .split(';')
+    .map((string) => string.trim().split(':'));
```

Comment:
The method 'split' does not mutate the original string, so there's no need to assign the result to a new variable. You can chain the subsequent methods directly.

Reply:
> The method 'split' does not mutate the original string, so there's no need to assign the result to a new variable. You can chain the subsequent methods directly.

Good point, you can just use methods in the chain

Reply:
I tried and at first I wrote everything in a line using the chain method, but with the lint command it moves to a new line.

Reply:
> I tried and at first I wrote everything in a line using the chain method, but with the lint command it moves to a new line.

It should be from the new line, but instead of creating so many variables such as `trimmedStyleDeclarations` you can do it in one chain

Reply:
Sorry, I am little confused, another mentor who watched and corrected my work recommended that I break the chain into several variables because it would be more clear.